src/assisted_installer_controller/assisted_installer_controller_test.go

```diff
@@ -1283,6 +1283,7 @@ var _ = Describe("installer HostRoleMaster role", func() {
 		BeforeEach(func() {
 			GeneralProgressUpdateInt = 100 * time.Millisecond
 			WaitTimeout = 150 * time.Millisecond
+			CVOMaxTimeout = 150 * time.Second
```

Comment:
While must tests shouldn't reach this timeout, no test should take more than a second so I think you can set it to:
```suggestion
			CVOMaxTimeout = time.Second
```

Reply:
i wanted to set it to something huge